test/Sentry.Unity.Tests/EmbeddedResourcesTests.cs

```diff
@@ -0,0 +1,18 @@
+using System.Linq;
+using System.Reflection;
+using NUnit.Framework;
+
+namespace Sentry.Unity.Tests
+{
+    public sealed class EmbeddedResourcesTests
+    {
+        [Test]
+        public void LinkXml_Embedded()
+        {
+            var resourceNames = Assembly.GetAssembly(typeof(SentryUnity)).GetManifestResourceNames();
+
+            Assert.NotNull(resourceNames);
+            Assert.IsTrue(resourceNames.Contains("Sentry.Unity.xml"));
```

Comment:
is there a:
```suggestion
            Assert.Contains(resourceNames, "Sentry.Unity.xml");
```
instead?

Or the order is inverted.
This might be xUnit only

Reply:
There was! Totally missed it. Thanks!